src/Service/Action.php

```diff
@@ -721,15 +721,20 @@ public function getOption(string $optionName)
      * @param array $arguments
      * @return ActionInterface
      */
-    public function setRpcQuery(string $method = null, array $arguments = []): ActionInterface
+    public function setRpcQuery(string $method = null, array $arguments = [], $id = null): ActionInterface
```

Comment:
I think `?string $id = null` may be better with `$id =null`.

Reply:
done.